worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -528,35 +546,66 @@ private[celeborn] class Worker(
     throw new CelebornException("Register worker failed.", exception)
   }
 
-  private def handleResourceConsumption(): util.Map[UserIdentifier, ResourceConsumption] = {
+  private def handleUserResourceConsumption(): util.Map[UserIdentifier, ResourceConsumption] = {
```

Comment:
This method handleUserResourceConsumption is similar to handleAppResourceConsumption. They can be merged.